lookout/style/typos/analyzer.py

```diff
@@ -1,6 +1,9 @@
 """Identifier typos analyzer."""
 from collections import defaultdict
+from functools import reduce
+from itertools import product
```

Comment:
Can you please use `itertools.product`, because "product" alone may mean many things. E.g. what we are building right now :)